files/scripts/asic_status.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+
+"""
+    bootstrap-asic
+"""
+try:
+    import re
+    import syslog
+    import sys
+    import threading
+    from sonic_py_common import daemon_base
+    from swsscommon import swsscommon
+except ImportError as e:
+    raise ImportError(str(e) + " - required module not found")
+
+#
+# Constants ====================================================================
+#
+CHASSIS_ASIC_INFO_TABLE = 'CHASSIS_ASIC_TABLE'
+SELECT_TIMEOUT_MSECS = 5000
+
+def main():
+    stop_event = threading.Event()
+    if len(sys.argv) != 2:
+        raise Exception('Pass valid asic-id as argument')
+
+    args_asic_id = sys.argv[1]
+
+    # Get platform from machine.conf
+    platform='unknown'
+    with open('/host/machine.conf') as f:
+        for line in f.readlines():
+            k, v = line.rstrip("\n").split("=")
+            if (k == 'onie_platform') or (k == 'aboot_platform'):
+                platform=v
+                break
+
+    if platform == 'unknown':
+        raise Exception('Unable to detect valid platform')
+
+    # Get num asics
+    num_asics=0
+    asic_file = '/usr/share/sonic/device/'+platform+'/asic.conf'
```

Comment:
There are API's we can use to get the asic.conf file https://github.com/Azure/sonic-buildimage/blob/96308314fb826f2f0653d912fd1b9d345e2cd2be/src/sonic-py-common/sonic_py_common/device_info.py#L144

Reply:
Thanks.